src/components/Card.js

```diff
@@ -25,6 +28,21 @@ function Card({ i, cardData, onSwipe, layer }) {
 
     const [isGoneState] = useState({ isGone: false })
 
+    const handleSwipe = direction => {
+        isGoneState.isGone = true
+        window.setTimeout(() => {
+            onSwipe(cardData, direction)
+        }, 100)
```

Comment:
As an unrelated discussion, I'm thinking about if we could change this:

```js
window.setTimeout(() => {
    onSwipe(cardData, direction)
}, 0)
```

to simply this:

```js
onSwipe(cardData, direction)
```

Reply:
The reason for the delay is to allow for the animation to continue slightly after a card has been swiped. Removing the the timeout will make the card disappear instantly which is not really a nice experience.

I would like to improve this at a later point but I think it might be more efficient to get rid of the delay when we experiment with entirely new card decision mechanics. Which is why I am quite fine with the current implementation.

Reply:
Yeah, we might have to decouple the swiping animation from the state update somehow. That way, state could be updated faster than the animation is completed.

But that will have to wait until the core simulation is in place